webservice/src/main/java/pl/edu/pjatk/lnpayments/webservice/auth/service/JwtService.java

```diff
@@ -62,4 +63,12 @@ public boolean isTokenValid(String token) {
         return false;
     }
 
+    static public Optional<String> headerToToken(String header) {
```

Comment:
Ta metoda nie musi być statyczna, wszędzie gdzie jest używana, wstrzyknięty jest JwtService. Statyczna metoda tutaj psuje enkapsulacje i zmniejsza wydajność (statyczne metody lecą do innej pamięci, która jest rzadziej czyszczona. Po drugie to konwencja mówi, że modyfikator dostępu idzie zawsze pierwszy, czyli powinno być `public static` jak już.